pkg/apis/pipeline/v1beta1/pipeline_validation_test.go

```diff
@@ -1500,7 +1501,7 @@ func TestValidatePipelineParameterVariables_Failure(t *testing.T) {
 		}},
 		expectedError: apis.FieldError{
 			Message: `"array" type does not match default value's type: "string"`,
-			Paths:   []string{"params[foo].default.type", "params[foo].type"},
+			Paths:   []string{"params.foo.default.type", "params.foo.type"},
```

Comment:
Maybe we don't want to change the existing tests - we want to add an extra sets of `params.foo.default.type` type of tests? Are we not supporting reference such as `params[foo].type` anymore?

Reply:
It's not reference. Just hint for path in the error message.

Reply:
AH I see. You can close this thread